libraries/AP_DAL/AP_DAL.h

```diff
@@ -0,0 +1,363 @@
+#pragma once
+
+#include <stdint.h>
```

Comment:
Should move these to be last in the include list

Reply:
done